lambdas/service/app.py

```diff
@@ -708,8 +709,8 @@ def validate_catalog(catalog):
         raise BRE(e)
```

Comment:
Is there a reason why we want to still return a 400 in this case? An invalid catalog is a sub-case of a nonexistent catalog, so the 404 would still be appropriate, and as a user, I wouldn't care if the catalog parameter I specified was "valid" according to the internal logic of the web service or not.

Reply:
Ah, nevermind, the test cases presented below clarify why the distinction is useful :+1: